### ISSUE TITLE
data preparation: formation of imdb

### ISSUE THREAD

**Issue by KevinChen1618** [vlfeat/matconvnet#216]
In implementation of  my own CNN, I wanted to import the data for classification tasks. My data consists of 982 images' features, where 80% (786 images) of them were used for training, and the other 20% (196 images) were used for testing. So my training data was of 786*1 cell array, and along with labels of  786*1 double array; Moreover, testing data was of 196*1 cell array, and along with labels of 196*1 double array. 

Then I put them it to the network, which was a duplicating structure from sample code "cnn_cifar", however, I got the following error messages:

Error using reshape
Product of known dimensions, 3072, not divisible into total number of elements, 786.

Error in cnn_LIVEII>getLIVEImdb (line 126)
    data{fi} = permute(reshape(fd.data',32,32,3,[]),[2 1 3 4]); % *this line

So, my question are that how could I properly set the data for training my own CNN, and what were these two error messages stand for?

**Comment by omegafil** [comment]
Hi,

did you check your images' dimensions? (reshape needs the correct size in your case)

**Comment by KevinChen1618** [comment]
Yes, I have checked the dimensions, and I have solved the problem. Thanks a lot though!
